### DISCUSSION TITLE
Different Pair Definitons for Different Exchanges

### DISCUSSION THREAD

**Discussion by mertdogan** [Drakkar-Software/OctoBot#1685] [Ideas]
Different exchanges serves different pairs. So, we may define different pairs at Octobot for different exchanges. Each exchange must have a setting for pairs, seperately. Common pair definition is not compatible for multiple exchange definition.

**Comment by Herklos** [comment]
> Different exchanges serves different pairs. So, we may define different pairs at Octobot for different exchanges. Each exchange must have a setting for pairs, seperately. Common pair definition is not compatible for multiple exchange definition.

Hi @mertdogan, it's a good idea, can you add it to https://feedback.octobot.online/ please ? Thanks !

**Reply by mertdogan** [reply]
Thank you for your reply. I added.
